src/js/layers/TreeCoverLossLayer.js

```diff
@@ -37,8 +37,8 @@ export default declare('TreeCoverLossLayer', [TileCanvasLayer], {
   },
 
   // Filter Data Method
-  filter: function(data) {
-    const z = brApp.map.getZoom();
```

Comment:
The report didn't know what `brApp` was, so I'm just passing the zoom level to the filter function now